File: Leetcode/python/sol771.py
# https://leetcode.com/problems/jewels-and-stones/
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        jewels = set((i for i in jewels))
        rslt = 0
        for i in stones:
            if i in jewels:
                rslt+=1

        return rslt

File: Leetcode/cpp/sol771.cpp

// https://leetcode.com/problems/jewels-and-stones/

class Solution {
public:

    int numJewelsInStones(string jewels, string stones) {
        set<char> unique(jewels.begin(), jewels.end());
        int count = 0;
        for(auto i:stones){
            if(unique.contains(i)){
                count++;
            }
        }
        return count;
    }
};